dune-project

```diff
@@ -96,7 +96,7 @@
   (graphics :with-test)
   (num :with-test)
   (ppx_expect (and (>= v0.14.2) :with-test))
-  (ppxlib (and (>= 0.15) :with-test))
```

Comment:
Why this change ?

Reply:
@talex5 @moyodiallo How can we find out why opam-dune-lint is complaining about this?

Reply:
By looking further, `ppxlib` is used by a private library `ppx_optcomp_light` in js_of_ocaml/compiler/ppx/dune.  As `js_of_ocaml_compiler` a public library in js_of_ocaml/compiler/lib/dune uses `ppx_optcomp_light` that makes `ppxlib` a build lib. This is why opam-dune-lint is complaining.

Reply:
`opam-dune-lint` is not supposed to complains because `js_of_ocaml_compiler` is a package/public-library it's where `ppxlib` is treated.

I'll try to fix it, Thanks.

Reply:
@smorimoto, I'm tempted to disable the checks in the CI for now, until the fix is released.
Also, I don't understand what makes `dune fmt` now complain. Is it just bumping the compiler to 5.1 ? or is it unrelated and just caused by a new version of dune ?

Reply:
It's because of the new version of dune which handle utf8 since 3.15.0.

https://github.com/ocaml/dune/releases/tag/3.15.0 
 - Dune file formatting: output utf8 if input is correctly encoded (https://github.com/ocaml/dune/pull/10113.